src/css/base.scss

```diff
@@ -124,8 +129,10 @@ h1 {
 }
 
 .my-bookings-wrapper {
-  @include flexbox(center, center);
+  display: flex;
```

Comment:
A suggestion: if you find you have a lot of display: flex and flex-direction, etc, you can put your flex properties into a mixin with variables and then use those throughout.